crypto/src/main/scala/coop/rchain/crypto/signatures/Secp256k1.scala

```diff
@@ -16,6 +15,8 @@ object Secp256k1 {
   private val provider  = new BouncyCastleProvider()
   private val curveName = "secp256k1"
 
+  private def toHex(bytes: Array[Byte]) = bytes.map(byte => "%02X".format(byte)).mkString
```

Comment:
We could move this to some utilities file. Where would be a proper location for such a function?